app/forms/user_form.rb

```diff
@@ -0,0 +1,25 @@
+class UserForm < Form
+  FIELDS = %i[
+    first_name
+    last_name
+    email
+    id
+  ].freeze
+
+  attr_accessor(*FIELDS)
+
+  validates :first_name, presence: true
+  validates :last_name, presence: true
+  validates :email, presence: true, format: { with: /\A.*@.*\z/ }
```

Comment:
Given that this will be used by the wider user sets

I think we should tighten the regex used and apply this regex 

https://github.com/alphagov/notifications-utils/blob/648e7c04641b6b68340e30ae1702f3e0ef6bf94d/notifications_utils/__init__.py#L14

wdyt @darokel

Reply:
https://github.com/DFE-Digital/publish-teacher-training/pull/3159/commits/fbdf4ad7636259c93d61195c737d5f19a17ad6d9